assets/css/edd-admin.css

```diff
@@ -468,7 +468,7 @@ label.edd_prices_shipping { /* remove once Simple Shipping markup is improved */
 	top: 3px;
 	right: 7px;
 }
-.edd_upload_field {
+input[type="text"].edd_upload_field {
 	padding-right: 8em;
```

Comment:
I actually think I'd prefer an `!important` here and reverting the selector back to `.edd_upload_field` . That way we won't get caught out if we ever change the input type for whatever reason (not that I'm planning to, but you never know :shrug: ).